common/views/components/Buttons/Control/Control.js

```diff
@@ -70,10 +76,25 @@ const Control = forwardRef(
     }
 
     return (
-      // $FlowFixMe (conditional JSX tag)
-      <HtmlTag ref={ref} {...attrs}>
-        <InnerControl text={text} icon={icon} />
-      </HtmlTag>
+      <>
+        {nextLink ? (
+          <NextLink
+            {...nextLink}
+            scroll={scroll}
```

Comment:
Not sure what is easier to read, but was thinking you can use this syntax:
```jsx 
<NextLink
            {...nextLink}
            {... {scroll, replace, prefetch }}
          />
```

Reply:
I think I'ma leave it as is unless you object – I think it looks nice, but I find it considerably harder to wrap my head around!

Reply:
No objection, just a thought.